src/platform_impl/apple/appkit/view.rs

```diff
@@ -801,6 +809,7 @@ impl WinitView {
             forward_key_to_app: Default::default(),
             marked_text: Default::default(),
             accepts_first_mouse,
+            focusable: focusable.into(),
```

Comment:
Nit: I prefer `Cell::new` explicitly.

```suggestion
            focusable: Cell::new(focusable),
```